src/main/java/edu/umich/its/cpm/Utils.java

```diff
@@ -152,8 +151,11 @@
 	public static final String REPORT_ATTR_ROLE = "role";
 	public static final String HAS_CONTENT_ITEM = "hasContentItem";
 	public static final String FILE_EXTENSION_BIN = "bin";
+    public static final String STATUS_DEPENDENCIES_BOX = "status/dependencies/box";
+    public static final String STATUS_DEPENDENCIES_CTOOLS = "status/dependencies/ctools";
+    public static final String REPORT_STATUS_DOWN = "DOWN";
```

Comment:
There are lots of possible errors that aren't the same as being down.  This should be more generic.  The status url Wiki page shows a slightly different format for up / error messages.